### PR TITLE
update model, created new migrations

### PR DESCRIPTION
**Pull request by andrewkorol**

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/elsa-workflows/elsa-core?pullRequest=1567) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/elsa-workflows/elsa-core?pullRequest=1567) before we can accept your contribution.<br/><br/>:x: andrewkorol [sign now](https://cla.dotnetfoundation.org/elsa-workflows/elsa-core?pullRequest=1567)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/elsa-workflows/elsa-core?pullRequest=1567) it.</sub>

File: src/modules/workflowsettings/Elsa.WorkflowSettings.Persistence.EntityFramework.MySql/Elsa.WorkflowSettings.Persistence.EntityFramework.MySql.csproj [hunk 1]
```diff
@@ -25,4 +25,8 @@
       <ProjectReference Include="..\Elsa.WorkflowSettings.Persistence.EntityFramework.Core\Elsa.WorkflowSettings.Persistence.EntityFramework.Core.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Folder Include="Migrations\" />
+    </ItemGroup>
```
Comment (sfmskywalker) [inline src/modules/workflowsettings/Elsa.WorkflowSettings.Persistence.EntityFramework.MySql/Elsa.WorkflowSettings.Persistence.EntityFramework.MySql.csproj:30]: Let's remove these lines.

File: src/modules/workflowsettings/Elsa.WorkflowSettings.Persistence.EntityFramework.PostgreSql/Elsa.WorkflowSettings.Persistence.EntityFramework.PostgreSql.csproj [hunk 2]
```diff
@@ -24,4 +24,8 @@
     <ProjectReference Include="..\Elsa.WorkflowSettings.Persistence.EntityFramework.Core\Elsa.WorkflowSettings.Persistence.EntityFramework.Core.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Migrations\" />
+  </ItemGroup>
+
```
Comment (sfmskywalker) [inline src/modules/workflowsettings/Elsa.WorkflowSettings.Persistence.EntityFramework.PostgreSql/Elsa.WorkflowSettings.Persistence.EntityFramework.PostgreSql.csproj:30]: Let's remove these lines also.

File: src/modules/workflowsettings/Elsa.WorkflowSettings.Persistence.EntityFramework.SqlServer/Elsa.WorkflowSettings.Persistence.EntityFramework.SqlServer.csproj [hunk 3]
```diff
@@ -25,4 +25,8 @@
         <ProjectReference Include="..\Elsa.WorkflowSettings.Persistence.EntityFramework.Core\Elsa.WorkflowSettings.Persistence.EntityFramework.Core.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Folder Include="Migrations\" />
+    </ItemGroup>
+
```
Comment (sfmskywalker) [inline src/modules/workflowsettings/Elsa.WorkflowSettings.Persistence.EntityFramework.SqlServer/Elsa.WorkflowSettings.Persistence.EntityFramework.SqlServer.csproj:31]: Remove.

File: src/modules/workflowsettings/Elsa.WorkflowSettings.Persistence.EntityFramework.Sqlite/Elsa.WorkflowSettings.Persistence.EntityFramework.Sqlite.csproj [hunk 4]
```diff
@@ -24,4 +24,8 @@
         <ProjectReference Include="..\Elsa.WorkflowSettings.Persistence.EntityFramework.Core\Elsa.WorkflowSettings.Persistence.EntityFramework.Core.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Folder Include="Migrations\" />
+    </ItemGroup>
+
```
Comment (sfmskywalker) [inline src/modules/workflowsettings/Elsa.WorkflowSettings.Persistence.EntityFramework.Sqlite/Elsa.WorkflowSettings.Persistence.EntityFramework.Sqlite.csproj:30]: Remove.

**Review by sfmskywalker — APPROVED** [review]

**Pull request merged**